README.md

```diff
@@ -95,6 +104,84 @@ optional arguments:
                         enable loading and storing SSO information from/to given directory
 ```
 
+### Docker Usage
+
+This section contains some tips and tricks to run the script using Docker. See the [Usage](#usage) section above for general script usage.
+
+#### Create working directory
+
+Create a directory where you want to store the garmin export data and session data:
+
+```shell
+mkdir -p ~/Documents/garmin_export
+cd ~/Documents/garmin_export
+```
+
+#### Set environment variables
+
+Export your username and password as environment variables:
+
+```shell
+export GARMIN_USERNAME=abc
+export GARMIN_PASSWORD=xyz
+```
+
+#### Run the docker container
+
+When running the docker container, make sure to:
+
+1. Override the `--directory` and `--session` command line arguments with static directories in the container.
+2. Mount the static `export_data` and `session_data` directories in the container to your local directories.
+
+This results in garmin export data being outputted to the local `./my_garmin_data` directory.
+
+```shell
+docker run -it --rm \
+  -v "$(pwd)/my_garmin_data:/export_data" \
+  -v "$(pwd)/my_garmin_session:/session_data" \
+  garmin-connect-exporter \
```

Comment:
@pe-st we need to replace this with the real image name in a follow-up PR

Reply:
Yepp: https://github.com/pe-st/garmin-connect-export/pull/122